integration_tests/test_runner/src/tests/lockup.rs

```diff
@@ -2,27 +2,29 @@ use std::time::SystemTime;
 
```

Comment:
This test still doesn't have IBC transfer testing because the test env does not currently have a gaia IBC test chain, I've made a task for that.